hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/codecs/ConsensusServiceStateTranslator.java

```diff
@@ -0,0 +1,98 @@
+                    monoTopics,
+            WritableKVState<TopicID, Topic> appTopics) {
+        com.hedera.node.app.service.mono.utils.MiscUtils.forEach(
+                com.hedera.node.app.service.mono.state.adapters.MerkleMapLike.from(monoTopics), (num, merkleTopic) -> {
+                    final var pbjTopic = stateToPbj(merkleTopic);
+                    appTopics.put(new TopicID(0, 0, pbjTopic.topicNumber()), pbjTopic);
+                });
+    }
+
+    @NonNull
+    public static Topic stateToPbj(@NonNull com.hedera.node.app.service.mono.state.merkle.MerkleTopic monoTopic) {
+        requireNonNull(monoTopic);
+        final var topicBuilder = new Topic.Builder();
+        topicBuilder.memo(monoTopic.getMemo());
+        if (monoTopic.hasAdminKey()) topicBuilder.adminKey(PbjConverter.asPbjKey(monoTopic.getAdminKey()));
+        if (monoTopic.hasSubmitKey()) topicBuilder.submitKey(PbjConverter.asPbjKey(monoTopic.getSubmitKey()));
+        topicBuilder.autoRenewPeriod(monoTopic.getAutoRenewDurationSeconds());
+        topicBuilder.autoRenewAccountNumber(monoTopic.getAutoRenewAccountId().num());
+        topicBuilder.expiry(monoTopic.getExpirationTimestamp().getSeconds());
+        topicBuilder.runningHash(Bytes.wrap(monoTopic.getRunningHash()));
+        topicBuilder.sequenceNumber(monoTopic.getSequenceNumber());
+        topicBuilder.deleted(monoTopic.isDeleted());
+        topicBuilder.topicNumber(monoTopic.getAutoRenewAccountId().num());
+
+        return topicBuilder.build();
+    }
+
+    @NonNull
+    public static com.hedera.node.app.service.mono.state.merkle.MerkleTopic pbjToState(
+            @NonNull TopicID topicID, @NonNull ReadableTopicStore readableTopicStore) {
+        requireNonNull(topicID);
+        requireNonNull(readableTopicStore);
+        final var optionalFile = readableTopicStore.getTopicLeaf(topicID);
+        return pbjToState(optionalFile.orElseThrow(() -> new IllegalArgumentException("Topic not found")));
+    }
+
+    @NonNull
+    public static com.hedera.node.app.service.mono.state.merkle.MerkleTopic pbjToState(@NonNull Topic topic) {
+        requireNonNull(topic);
+        final com.hedera.node.app.service.mono.state.merkle.MerkleTopic monoTopic =
+                new com.hedera.node.app.service.mono.state.merkle.MerkleTopic(
+                        topic.memo(),
+                        (JKey) com.hedera.node.app.service.mono.pbj.PbjConverter.fromPbjKey(
+                                        topic.adminKeyOrElse(Key.DEFAULT))
+                                .orElse(null),
+                        (JKey) com.hedera.node.app.service.mono.pbj.PbjConverter.fromPbjKey(
+                                        topic.submitKeyOrElse(Key.DEFAULT))
+                                .orElse(null),
+                        topic.autoRenewPeriod(),
+                        new com.hedera.node.app.service.mono.state.submerkle.EntityId(
```

Comment:
I believe `MerkleTopic` expects the `autoRenewAccountId` to be `null` if it is not set, and not `EntityId(0, 0, 0)`.